usermods/matter_over_wifi/setup_matter_component.py

```
@@ -0,0 +1,34 @@
+
+# ── setup_matter_component.py ─────────────────────────────────────────────────
+# Pre-build script for the esp32s3_matter_wifi environment.
+#
+# Copies the idf_component.yml manifest from the Matter usermod into wled00/
+# so that pioarduino's built-in ComponentManager (and the IDF Component Manager)
+# can resolve the espressif/esp_matter dependency.
+#
+# The destination file is listed in .gitignore and must NOT be committed.
+# ─────────────────────────────────────────────────────────────────────────────
+
+project_dir = Path(env["PROJECT_DIR"]).resolve()
```

Ruff (0.15.7)

[error] Undefined name `env`

(F821)